app/assets/stylesheets/utilities/_overrides.scss

```diff
@@ -102,3 +102,7 @@
   }
 }
 // scss-lint:enable IdSelector
+
+.gem-c-inset-text {
```

Comment:
Where is this being used?

Reply:
It's on the `inset-text` component we pull into `show.html`.

Reply:
Can this changed be added where it's used to make that clear?